e2e/cypress/support/ui_commands.js

```diff
@@ -60,14 +60,22 @@ Cypress.Commands.add('uiChangeMessageDisplaySetting', (setting = 'STANDARD') =>
 
 // Type Cmd or Ctrl depending on OS
 Cypress.Commands.add('typeCmdOrCtrl', () => {
+    typeCmdOrCtrlInt('#post_textbox');
+});
+
+Cypress.Commands.add('typeCmdOrCtrlForEdit', () => {
+    typeCmdOrCtrlInt('#edit_textbox');
+});
+
+function typeCmdOrCtrlInt(textboxSelector) {
     let cmdOrCtrl;
     if (isMac()) {
         cmdOrCtrl = '{cmd}';
     } else {
         cmdOrCtrl = '{ctrl}';
     }
 
-    cy.get('#post_textbox').type(cmdOrCtrl, {release: false});
+    cy.get(textboxSelector).type(cmdOrCtrl, {release: false});
 });
```

Comment:
I think this should be
```suggestion
}
```